src/gui/folder.cpp

```diff
@@ -494,7 +494,7 @@ void Folder::slotRunEtagJob()
     }
 
     if (!canSync()) {
-        qCInfo(lcFolder) << "Not syncing.  :" << remoteUrl().toString() << _definition.paused << AccountState::stateString(_accountState->state());
+        qCInfo(lcFolder) << "Not syncing.  :" << remoteUrl().toString() << _definition.paused << _accountState->state();
```

Comment:
Do we really want to expose Qt's rendering of the enum's values in user-facing log messages?

Reply:
we do it for everything else.
Additionally there is not really anything like user facing logs.